batch_convert.qml

```diff
@@ -31,12 +31,12 @@ MuseScore {
       Qt.quit()
       }
     }
-
-//Settings {
-  //id: mscoreSettings
-  //category: "ui/application"
-  //property var globalStyle
-  //}
+  
+  Settings {
+    id: mscorePathsSettings
+    category: "application/paths"
+    property var myScores
+    }
```

Comment:
Not quite sure, but I seem to remember that this part had been commented out in the mac branch for a reason, like because it didn't work

Reply:
Let me try it again.
I think the part that didn't work was setting the window color. [lines 50 to 53](https://github.com/Jojo-Schmitz/batch_export/pull/6/files/2516b695c61cf627d67d1ece68725ac42da2db8f#diff-8108c8b073573ef2b164b9f1f4873723L50)
But anyway the theme is followed by default.
I'll tell you monday afternoon.

Reply:
Ok, doesn't work on mac. Works on windows though... I guess this is why this is the mac branch :)